### DISCUSSION TITLE
Supported minecraft version

### DISCUSSION THREAD

**Discussion by Wither00000** [pterodactyl/panel#4848] [General Community]
On the documentation of Pterodactyl they said that it does supported Spigot, Sponge, Bungeecord, Waterfall, and more.

What other version does they support ? because "more" is not a version.

**Comment by parkervcp** [comment]
Do you know how many minecraft forks there are?

**Reply by Wither00000** [reply]
Just want to know if there is bukkit, fabric, forge, mohist, cauldron, quilt, cauldron, glowstone, arclight just these forks because they are popular not like bungeecord.

**Reply by parkervcp** [reply]
Those would be the 'and more' called out there.

**Reply by Wither00000** [reply]
Yes I want to know if the project support mohist or cauldron.
